docs/BreakingChanges/094 XmlSerializer serializes fields differently in 4.5.md

```diff
@@ -0,0 +1,39 @@
+## 94: XmlSerializer serializes fields differently in 4.5
```

Comment:
{
  "Id": "94",
  "Title": "XmlSerializer serializes fields differently in 4.5",
  "Details": "Changes in the XmlSerializer in .NET Framework 4.5 caused fields to be formatted differently in the serialized XML.",
  "Suggestion": "This behavior was corrected in a servicing update of .NET Framework 4.5. Please update the .NET Framework 4.5, or upgrade to .NET Framework 4.5.1 or later, to fix this issue. Alternatively, the following config setting will revert to 4.0 XmlSerializer behavior:\n\n```xml\n<system.xml.serialization>\n<xmlSerializer useLegacySerializerGeneration=\"true\" />\n</system.xml.serialization>\n```",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/761786/net-4-5-xmlserializer-produces-different-output-from-4-0",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.Stream,System.Object)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.TextWriter,System.Object)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Object,System.Xml.Serialization.XmlSerializationWriter)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.Stream,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.TextWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces,System.String)",
    "M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces,System.String,System.String)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 1
}

**Parsed APIs:**
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.Stream,System.Object)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.TextWriter,System.Object)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.Object,System.Xml.Serialization.XmlSerializationWriter)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.Stream,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.IO.TextWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces,System.String)
- [x] M:System.Xml.Serialization.XmlSerializer.Serialize(System.Xml.XmlWriter,System.Object,System.Xml.Serialization.XmlSerializerNamespaces,System.String,System.String)